plugins/WebsiteType/lang/en.json

```diff
@@ -0,0 +1,7 @@
+{
+  "WebsiteType": {
+    "Website": "Website",
+    "Websites": "Websites",
+    "WebsiteDescription": "A website or a mobile web app that is usually consumed via a browser"
```

Comment:
Maybe `A website consists of web pages typically served from a single web domain.` ?